packages/lib-content/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "@zooniverse/content",
```

Comment:
Definitely open to renaming this to `content-components` or similar if we want to!